lib/internal/utils.js

```diff
@@ -107,35 +107,52 @@ module.exports = {
      * @returns {{kind: string, value: string, description: string | null}|null} The parsed data of the given comment. If `null`, it is not a directive comment.
      */
     parseDirectiveComment(comment) {
-        const { text, description } = divideDirectiveComment(comment.value)
-        const match = DIRECTIVE_PATTERN.exec(text)
-
-        if (!match) {
+        const parsed = parseDirectiveText(comment.value)
+        if (!parsed) {
             return null
         }
-        const directiveText = match[1]
-        const lineCommentSupported = LINE_COMMENT_PATTERN.test(directiveText)
+
+        const lineCommentSupported = LINE_COMMENT_PATTERN.test(parsed.kind)
 
         if (comment.type === "Line" && !lineCommentSupported) {
             return null
         }
 
         if (
-            lineCommentSupported &&
+            parsed.kind === "eslint-disable-line" &&
```

Comment:
Multi-line comments seem to be allowed except for `eslint-disable-line`.
https://github.com/eslint/eslint/pull/15436